internal/bminventory/inventory_test.go

```diff
@@ -8899,44 +8899,6 @@ var _ = Describe("KubeConfig download", func() {
 		Expect(*replyPayload.URL).Should(Equal("url"))
 	})
 
-	It("kubeconfig download no cluster id", func() {
-		clusterId := strToUUID(uuid.New().String())
-		generateReply := bm.DownloadClusterKubeconfig(ctx, installer.DownloadClusterKubeconfigParams{
-			ClusterID: *clusterId,
-		})
-		Expect(generateReply).To(BeAssignableToTypeOf(&common.ApiErrorResponse{}))
-		Expect(generateReply.(*common.ApiErrorResponse).StatusCode()).To(Equal(int32(http.StatusNotFound)))
-	})
-	It("kubeconfig download cluster is not in installed state", func() {
-		generateReply := bm.DownloadClusterKubeconfig(ctx, installer.DownloadClusterKubeconfigParams{
-			ClusterID: clusterID,
-		})
-		Expect(generateReply).To(BeAssignableToTypeOf(&common.ApiErrorResponse{}))
-		Expect(generateReply.(*common.ApiErrorResponse).StatusCode()).To(Equal(int32(http.StatusConflict)))
-	})
-	It("kubeconfig download s3download failure", func() {
-		status := models.ClusterStatusInstalled
-		c.Status = &status
-		db.Save(&c)
-		fileName := fmt.Sprintf("%s/%s", clusterID, constants.Kubeconfig)
-		mockS3Client.EXPECT().Download(ctx, fileName).Return(nil, int64(0), errors.Errorf("dummy"))
-		generateReply := bm.DownloadClusterKubeconfig(ctx, installer.DownloadClusterKubeconfigParams{
-			ClusterID: clusterID,
-		})
-		Expect(generateReply.(*common.ApiErrorResponse).StatusCode()).To(Equal(int32(http.StatusConflict)))
-	})
-	It("kubeconfig download happy flow", func() {
```

Comment:
Leave the happy flow to test the 404 response (or any minimal scenario to that effect)

Reply:
why?
we remove `DownloadClusterKubeconfig` from the code. and it will get completely removed once we also remove this from swagger.
I added a test that uses `V2GetCredentials` instead, so we didn't lose coverage.

Reply:
We don't remove the API from swagger just in order to make to many conflicts so it's not something that we expect to keep, i don't think that we need to check 404 for the api

Reply:
in the next release we will not have the api at all.